src/Makefile.am

```diff
@@ -17,6 +17,21 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+
+gresource_xml  := $(top_srcdir)/data/io.github.TransmissionRemoteGtk.gresource.xml
```

Comment:
I believe this should be added to `EXTRA_DIST` (as in `make distcheck` probably fails atm).

Reply:
Good catch, RESOURCE_FILES as well